src/channels/components/ChannelForm/MarkAsPaid.tsx

```diff
@@ -7,20 +7,20 @@ import { FormattedMessage } from "react-intl";
 import { messages } from "./messages";
 
 interface MarkAsPaidProps {
-  isDiabled: boolean;
```

Comment:
question: why did you remove disabled from props IMO is quite important when loading data?

Reply:
Because none of the checkboxes in this form have it. Should we disable all checkboxes on loading?